docs/deployment.md

```diff
@@ -76,7 +89,12 @@ or, if you don't have the source code
 ```
 docker run -it --rm jaegertracing/jaeger-collector /go/bin/collector-linux -h
 ```
-
+Example:
+```
+docker run -it --rm -p14267:14267 -p14268:14268
```

Comment:
This should either be all in one line, or there should be a `\` at the end of the line, so that readers can just copy/paste from the instructions to the terminal.